### DISCUSSION TITLE
Core-7.7 not included when I do a curl

### DISCUSSION THREAD

**Discussion by kgermann-harris** [cobbler/cobbler#2583] [Q&A]
### Version

```
Cobbler 2.8.5
  source: ?, ?
  build time: Tue Oct 15 01:59:43 2019
```
### Question
`curl "http://x.x.x.x/cblr/svc/op/yum/profile/Workstation-CentOS7.7-UEFI" --output /tmp/cobbler-config.repo `

The repo file in /var/www/cobbler/ks_mirror/config for CentoOS-7.7 that is in the profile is not being included in the cobbler-config.repo file. 

This stanza is not included with this curl like it should. I need to know what I missing in the cobbler configuration. It works for 7.5.
```
[core-7.7]
name=core-7.5
baseurl=http://@@http_server@@/cobbler/ks_mirror/CentOS-7.7-1908-Everything-x86_64
enabled=1
gpgcheck=0
priority=$yum_distro_priority
```

**Comment by kgermann-harris** [comment]
X.X.X.X is done on purpose due to requirements to obfuscate any company information :)

**Comment by kgermann-harris** [comment]
```
[root@xxxx config]# cd /var/www/cobbler/ks_mirror/config; ls
CentOS-7.5-1804-Everything-x86_64-0.repo
CentOS-7.7-1908-Everything-x86_64-1.repo
```

**Comment by SchoolGuy** [comment]
I would suggest starting to compare the distro, profile and system records.

As a starting point how you do this I would recommend this: https://cobbler.readthedocs.io/en/latest/cobbler.html#cobbler-report

**Comment by kgermann-harris** [comment]
I have looked in the cobbler reports. The /var/www/cobbler/ks_mirror/config/CentOS-7.7-1908-Everything-x86_64-1.repo
 file is not being included in the curl grab for the repo file. I am not sure what I am doing wrong in the cobbler setup to make sure that gets included with the profile using curl. I am not able to find any documentation that talks specifically on how to set this up. Are these repo files in /var/www/cobbler/ks_mirror/config automatically generated by the cobbler sync?

curl "http://172.x.x.x/cblr/svc/op/yum/profile/Workstation-CentOS7.7-UEFI" --output /tmp/cobbler-config.repo

This is the stanza I am trying to configure to be included with the curl grab. 
```
[core-7.7]
name=core-7.7
baseurl=http://@@http_server@@/cobbler/ks_mirror/CentOS-7.7-1908-Everything-x86_64
enabled=1
gpgcheck=0
priority=$yum_distro_priority
```
The 7.5 version of this is working when I do the curl for CentOS7.5. 

I looked at the cobbler reports as you suggest. I am not seeing whats missing. Your help is appreciated.

**Comment by SchoolGuy** [comment]
I have to honestly say I never have set this up myself or used the feature. I will try to have a look as soon as I got time.

**Comment by kgermann-harris** [comment]
I am trying to get this so it produces the [core-7.7] report for the Everything ISO distro.

curl "http://@@http_server@@/cblr/svc/op/yum/profile/Workstation-CentOS7.7-UEFI" --output /etc/yum.repos.d/cobbler-config.repo

This produces:
```
[core-7.5]
name=core-7.5
baseurl=http://172.24.79.197/cobbler/ks_mirror/CentOS-7.5-1804-Everything-x86_64
enabled=1
gpgcheck=0
priority=1
```
This should be the 7.7 distro/repo. 

I appreciate your help.
